src/main/java/Lotes.java

```diff
@@ -0,0 +1,108 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+public class Lotes {
+    private static final ArrayList<Task> taskList = new ArrayList<>();
+    private static final String lineSeparator = System.lineSeparator();
+    private static final String horizontalLine = "    _________________________"
+                                                + "___________________________";
+
+    private static void printTasksList() {
+        System.out.println(horizontalLine);
+
+        if (taskList.isEmpty()) {
+            System.out.println("     List is empty, please enter some text to add to list.");
+        } else {
+            System.out.println("     Here are the tasks in your list:");
+
+            int index = 0;
+            for (Task task : taskList) {
+                index++;
+                System.out.print("     " + index + ".[" + task.getStatusIcon()
+                                + "] " + task.getDescription() + lineSeparator);
+            }
+        }
+        System.out.println(horizontalLine);
+    }
+
+    private static void markTask(String userInput) {
+        try {
+            int number = Integer.parseInt(userInput.substring(5))
+                    - 1;
+
+            taskList.get(number).setDone(true);
+
+            System.out.print(horizontalLine + lineSeparator
+                    + "     Nice! I've marked this task as done:"
+                    + lineSeparator + "     [" + taskList.get(number).getStatusIcon()
+                    + "] " + taskList.get(number).getDescription()
+                    + lineSeparator + horizontalLine + lineSeparator);
+
+        } catch (IndexOutOfBoundsException | NumberFormatException s) {
+            System.out.println("     Invalid integer input");
+            printTasksList();
+        }
+    }
+
+    private static void unMarkTask(String userInput) {
+        try {
+            int taskListIndex = (Integer.parseInt(userInput.substring(7))
+                    - 1);
+
+            taskList.get(taskListIndex).setDone(false);
+
+            System.out.print(horizontalLine + lineSeparator
+                    + "     OK, I've marked this task as not done yet:" + lineSeparator
+                    + "     [" + taskList.get(taskListIndex).getStatusIcon() + "] "
+                    + taskList.get(taskListIndex).getDescription() + lineSeparator
+                    + horizontalLine + lineSeparator);
+
+        } catch (IndexOutOfBoundsException | NumberFormatException s) {
```

Comment:
Suggestion: Can consider separating the exceptions into separate catch staments so that you can better handle each exception individually. 

Alternatively, you can consider scoping the tasks into its own try-catch block so that the scope of exception is limited to that task. For example, instead of 

``` java
try {
    // Code snippet "a" 
    // Code snippet "b"
} catch (/* Exceptions thrown by "a" and/or "b" */) {
    // Catch Code
}
```


you can do

``` java
try {
    // Code snippet "a" 
} catch (/* Exceptions thrown by "a" */) {
    // Catch Code
}
// Some code
try {
    // Code snippet "b"
} catch (/* Exceptions thrown by "b" */){ 
    // Catch Code
}
```